apps/portal/src/content/docs/components/button-layout.mdx

```diff
@@ -0,0 +1,247 @@
+---
+title: ButtonLayout
+description: ButtonLayout component
+beta: true
+---
+
+The `ButtonLayout` component allows you to group multiple buttons together, either horizontally or vertically. It is useful for organizing related actions.
+
+import { DocsPage } from "../../../components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  contentClassName={"!block ![place-items:normal]"}
+  code={`
+    <ButtonLayout>
+      <Button variant="outline">Button outline</Button>
+      <Button variant="secondary">Button secondary</Button>
+      <Button>Button primary</Button>
+    </ButtonLayout>
+  `}
+/>
+
+## Usage
+
+```typescript jsx
+import { Button, ButtonLayout } from '@harnessio/ui/components'
+
+return (
+  <ButtonLayout horizontalAlign="start">
+    <Button>Button primary</Button>
+    <Button variant="secondary">Button secondary</Button>
+    <Button variant="outline">Button outline</Button>
+  </ButtonLayout>
+
+  <ButtonLayout orientation="vertical">
+    <Button>Button primary</Button>
+    <Button variant="secondary">Button secondary</Button>
+    <Button variant="outline">Button outline</Button>
+  </ButtonLayout>
+
+  <ButtonLayout.Root>
+    <ButtonLayout.Primary>
+      <Button>Button primary</Button>
+    </ButtonLayout.Primary>
+    <ButtonLayout.Secondary>
+      <Button variant="secondary">Button secondary</Button>
+    </ButtonLayout.Secondary>
+  </ButtonLayout.Root>
+)
+```
+
+## Anatomy
+
+All parts of the `ButtonLayout` component can be imported and composed as required.
+
+```typescript jsx
+import { ButtonLayout } from '@harnessio/ui/components'
+
```

Comment:
We can remove the import from the "Anatomy" section since we've touched on it in the "Usage" section